src/brokers/broker-details/BrokerDetails.container.tsx

```diff
@@ -20,8 +20,9 @@ import { useTranslation } from '../../i18n';
 import {
   ClientsContainer,
   ConfigurationContainer,
-  QueuesContainer,
-  TopicsContainer,
+  AddressContainer,
+  // QueuesContainer,
+  // TopicsContainer,
```

Comment:
It's better to not leave commented lines

Reply:
yes, i will remove and update the PR.
Thanks!